docs/release/release-tasks.md

```diff
@@ -373,6 +374,30 @@ Stakeholders are: (TBD)
 * Provider implementers
 * ...
 
+#### Communicate beta to providers
+
+The goal of this task is to inform all providers that a new beta.0 version a release is out and that it should be tested. We want to prevent issues where providers don't have enough time to test before a new version of CAPI is released. This stems from a previous issue we are trying to avoid: https://github.com/kubernetes-sigs/cluster-api/issues/8498
+
+We should inform at least the following providers via a new issue on their respective repos that a new version of CAPI is being released (provide the release date) and that the beta.0 version is ready for them to test.
+
+ - Addon provider helm: https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/issues/new/choose
```

Comment:
If you want to avoid the selection (which is likely as you probably you want to post the same issue everywhere): https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/issues/new